frontend/src/store/controller/selectors.ts

```diff
@@ -233,6 +234,14 @@ export const getPopupEventLog = createSelector(getController, (states: Controlle
   return states.eventLog.nextLogEvents.elements.find((el: LogEvent) => el.id === maxId);
 });
 
+// Backend Initialized
+export const getBackendInitialized = createSelector(
+  getSensorMeasurements,
+  getBackendDisconnected,
+  (sensorMeasurements: SensorMeasurements | null, backendDisconnected: boolean): boolean =>
+    sensorMeasurements !== null && backendDisconnected,
```

Comment:
Based on function names, the logic expressed in this selector is that the backend is initialized if both of the following conditions are true:
- sensorMeasurements is not-null
- the backend is disconnected.
Based on this, and how you implemented the `getBackendDisconnected` selector, maybe you actually meant to name the selector `getBackendConnected` rather than `getBackendDisconnected`?

Reply:
While we're at it, let's change `sensorMeasurements !== null` to `parametersRequest !== null && alarmLimitsRequest !== null` in this PR.